_inc/client/state/action-types.js

```diff
@@ -142,3 +142,6 @@ export const JETPACK_SITE_VERIFY_GOOGLE_REQUEST = 'JETPACK_SITE_VERIFY_GOOGLE_RE
 export const JETPACK_SITE_VERIFY_GOOGLE_REQUEST_SUCCESS =
 	'JETPACK_SITE_VERIFY_GOOGLE_REQUEST_SUCCESS';
 export const JETPACK_SITE_VERIFY_GOOGLE_REQUEST_FAIL = 'JETPACK_SITE_VERIFY_GOOGLE_REQUEST_FAIL';
+
+export const SITE_CHECKLIST_RECEIVE = 'SITE_CHECKLIST_RECEIVE';
```

Comment:
Should we include `PLAN` in the constant names, just in case we add more checklists in the future?

Reply:
🤔 I can make the change but I'm not sure there will be any conflict or that we need to address it preemptively.

In implementing this, I was thinking about receiving checklist data from the API. There's one checklist endpoint and it returns a bunch of data, one part  of it is a tasks object.

If we to want different names for these Redux action types in the future, it's should be pretty straightforward to change the name in a few files.

Reply:
That works for me 👍